docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -270,54 +345,39 @@ In addition to the ``namespace.collection_name:version`` format, you can provide
+Verifying signed collections
+-----------------------------
 
-.. code-block:: bash
+If a collection has been signed by a :term:`self-hosted Galaxy` server, the server will provide ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection's contents. This option is not available on community Galaxy. See :ref:`installing_signed_collections` for how to verify a signed collection when you install it.
 
-   gpg --import --no-default-keyring --keyring ~/.ansible/pubring.kbx my-public-key.asc
-
-In addition to any signatures provided by the Galaxy server, signature sources can also be provided in the requirements file and on the command line. Signature sources should be URIs.
-
-You can manually generate detached signatures for a collection using the ``gpg`` CLI using the following step. This step assume you have generated a GPG private key, but do not cover this process.
+To verify a signed installed collection:
 
 .. code-block:: bash
 
-   ansible-galaxy collection build
-   tar -Oxzf namespace-name-1.0.0.tar.gz MANIFEST.json | gpg --output namespace-name-1.0.0.asc --detach-sign --armor --local-user email@example.com -
+   ansible-galaxy collection verify my_namespace.my_collection  --keyring ~/.ansible/pubring.kbx
+
 
 Use the ``--signature`` option to verify collection name(s) provided on the CLI with an additional signature. This option can be used multiple times to provide multiple signatures.
 
 .. code-block:: bash
 
    ansible-galaxy collection verify my_namespace.my_collection --signature https://examplehost.com/detached_signature.asc --signature file:///path/to/local/detached_signature.asc --keyring ~/.ansible/pubring.kbx
 
-Collections in a requirements file should list any additional signature sources following the collection's "signatures" key.
-
-.. code-block:: yaml
-
-   # requirements.yml
-   collections:
-     - name: ns.coll
-       version: 1.0.0
-       signatures:
-         - https://examplehost.com/detached_signature.asc
-         - file:///path/to/local/detached_signature.asc
+Optionally, you can verify a collection signature with a ``requirements.yml`` file.
 
 .. code-block:: bash
 
    ansible-galaxy collection verify -r requirements.yml --keyring ~/.ansible/pubring.kbx
 
-When a collection is installed from a Galaxy server, the signatures provided by the server to verify the collection's authenticity are saved alongside the installed collections. This data is used to verify the internal consistency of the collection without querying the Galaxy server again when the ``--offline`` option is provided.
+When a collection is installed from a self-hosted Galaxy server, the signatures provided by the server to verify the collection's authenticity are saved alongside the installed collections. This data is used to verify the internal consistency of the collection without querying the self-hosted Galaxy server again when the ``--offline`` option is provided.
```

Comment:
self-hosted doesn't cover Automation Hub. I liked the original changes you made to this section.

Reply:
I'm less inclined to keep talking about AH here in the community docs as they have their own customer docs now. I can check with that team to see if they have this covered over on access.redhat.com yet tho

Reply:
Isn't it general ansible-galaxy documentation though, which is not community-specific?

Reply:
What do you think about `Pulp 3 (v3) Galaxy server` instead of `self-hosted Galaxy server`? That would keep it general. We could have a link to https://docs.pulpproject.org/pulp_ansible/ from the glossary.

Reply:
Hmm good point on this being ansible-galaxy documentation that is likely not on access.redhat.com. I'll add AH back.  I'm I'm not sure if the pulp docsite is better/clearer than pointing to the GalaxyNG docsite that this PR does now.

So the original commit had `Automation Hub or self-hosted pulp or on-premise Hub server`. That to me misses out on galaxyNG (oh how I wish I could just call it galaxy NG :-).

So I'll ask the AH writing team if I can just call it Automation Hub, and then we can decide what to call the galaxyNG version.  Can you explain why Pulp 3 (v3) Galaxy is better than self-hosted Galaxy?

Reply:
Pulp 3 (v3) covers AH and Galaxy NG. Perhaps it is not helpful in adding clarity to user docs, but it's more accurate. `ansible-galaxy` doesn't care whether a server is AH/NG/neither, it just needs to follow the v3 spec https://docs.pulpproject.org/pulp_ansible/restapi.html to have ansible-galaxy signature functionality.

Reply:
Ends up the AH on-prem doesn't fully support this feature, though when it does, the official name is Private Automation Hub.  That said, this is being documented separately for customers so I'll focus this as much as possible on Galaxy NG, which we can call Pulp 3 Galaxy.

Reply:
ok as I edit, I realize we already created a 'common term' for all of these besties  - a distribution server. So I'll go with  that and update the docs to match (as well as the table that lists the current options). More to come.